pkg/lightning/config/config_test.go

```diff
@@ -445,6 +445,17 @@ func TestAdjustSecuritySection(t *testing.T) {
 			hasTLS:         false,
 			fallback2NoTLS: false,
 		},
+		{
+			input: `
+				[security]
+				ca-path = "/path/to/ca2.pem"
```

Comment:
can you also add a unit test to check `security.ca-path` (`cfg.Security`) is unchanged? The new unit test should fail with the old code and should pass with this new PR.

Reply:
https://github.com/pingcap/tidb/pull/54211/files#diff-ebcda57fdf1bea10676a21181bc388b6dd8f95c8a2013438fd826cfec040e4c7R479-R490
I add a unit test. 
I did a comparison of the ca-path before and after adjusting, was this the right way to do it?

Reply:
I'll check it later. We have some flaky tests and they causes the CI failure, don't worry.